.github/workflows/ci.yml

```
@@ -651,21 +651,20 @@ jobs:
-          ROR_E2E_REPORTS_STORE_ACCESS_KEY_SECRET: ${{ secrets.ROR_E2E_REPORTS_STORE_ACCESS_KEY_SECRET }}
-          ROR_E2E_REPORTS_STORE_BUCKET: ${{ vars.ROR_E2E_REPORTS_STORE_BUCKET || secrets.ROR_E2E_REPORTS_STORE_BUCKET }}
-          ROR_E2E_REPORTS_STORE_REGION: ${{ vars.ROR_E2E_REPORTS_STORE_REGION || secrets.ROR_E2E_REPORTS_STORE_REGION }}
-          ROR_E2E_REPORTS_STORE_ENDPOINT_URL: ${{ vars.ROR_E2E_REPORTS_STORE_ENDPOINT_URL || secrets.ROR_E2E_REPORTS_STORE_ENDPOINT_URL }}
-          E2E_REPORTS_PATH_PREFIX: ${{ vars.ROR_E2E_REPORTS_STORE_PATH_PREFIX || secrets.ROR_E2E_REPORTS_STORE_PATH_PREFIX }}
+          ROR_S3_ENDPOINT_URL: ${{ vars.ROR_S3_ENDPOINT_URL }}
+          ROR_S3_REGION: ${{ vars.ROR_S3_REGION }}
+          ROR_S3_BUCKET: ${{ vars.ROR_S3_BUCKET }}
+          ROR_S3_ACCESS_KEY_ID: ${{ secrets.ROR_S3_ACCESS_KEY_ID }}
+          ROR_S3_SECRET_ACCESS_KEY: ${{ secrets.ROR_S3_SECRET_ACCESS_KEY }}
+          ROR_S3_PATH_E2E_REPORTS: ${{ vars.ROR_S3_PATH_E2E_REPORTS }}
         run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2155:warning:5:8: Declare and assign separately to avoid masking return values

(shellcheck)